app/src/main/java/com/android/sample/model/recipe/RecipeBuilder.kt

```diff
@@ -7,11 +7,11 @@ import com.android.sample.resources.C.Tag.ERROR_STR_THUMBNAIL
 
 /** Builder class for creating a Recipe instance. */
 class RecipeBuilder {
-  private var idMeal: String = ""
-  private var strMeal: String = ""
-  private var strCategory: String? = null
-  private var strArea: String? = null
-  private var strInstructions: String = ""
+  private var uid: String = ""
+  private var name: String = ""
+  private var category: String? = null
+  private var origin: String? = null
+  private var instructions: String = ""
   private var strMealThumbUrl: String = ""
```

Comment:
I noticed that strMealThumbUrl keeps the str prefix. Is there a reason for this?